octobot_trading/exchanges/util/exchange_market_status_fixer.py

```diff
@@ -259,6 +259,21 @@ def _fix_market_status_limits_with_price(self):
         price_max = limits_price.get(Ecmsc.LIMITS_PRICE_MAX.value)
         if not is_ms_valid(price_max, zero_valid=False):
             price_max = self.price_example * self.LIMIT_PRICE_MULTIPLIER
+        limit_cost = self.market_status[Ecmsc.LIMITS.value].get(Ecmsc.LIMITS_COST.value, {})
+        cost_min = limit_cost.get(Ecmsc.LIMITS_COST_MIN.value)
+        if not is_ms_valid(cost_min, zero_valid=False):
+            if (
+                is_ms_valid(limits_price.get(Ecmsc.LIMITS_PRICE_MIN.value))
+                and is_ms_valid(limits_amount.get(Ecmsc.LIMITS_AMOUNT_MIN.value))
+            ):
+                # min cost can be computed from min price and amount: those values come from exchange data
+                cost_min = limits_price.get(Ecmsc.LIMITS_PRICE_MIN.value) * limits_amount.get(Ecmsc.LIMITS_AMOUNT_MIN.value)
```

Comment:
I updated this part to be sure not to create invalid values

Reply:
Could we test that I does not create invalid values please?

Reply:
Right, up !

Reply:
I don't understand as I don't see any news test

Reply:
tests added